sagemaker_processing/local_pyspark/local_pyspark_example.ipynb

```diff
@@ -0,0 +1,532 @@
+{
```

Comment:
<p>Optionally add a Conclusion section summarizing what the customer has accomplished and suggesting any next steps.</p><p> </p><p>Confirming that there are no resources that continue incurring cost that need to be deleted? I believe not, since there's no deployed endpoint, etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3421/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>